packages/date-picker/src/FilterDateRangePicker/utils/getDateRangeDisplayLabel.spec.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react"
+import { render } from "@testing-library/react"
+import { enAU } from "date-fns/locale"
+import { getDateRangeDisplayLabel } from "./getDateRangeDisplayLabel"
+
+const locale = enAU
+
+describe("getDateRangeDisplayLabel", () => {
+  it("returns undefined if date range is not defined", () => {
+    expect(getDateRangeDisplayLabel(undefined, locale)).toBeUndefined()
+  })
+
+  it("returns undefined if date range is partially defined", () => {
+    expect(
+      getDateRangeDisplayLabel({ from: new Date("2022-05-01") }, locale)
+    ).toBeUndefined()
+    expect(
+      getDateRangeDisplayLabel(
+        { from: undefined, to: new Date("2022-05-01") },
+        locale
+      )
+    ).toBeUndefined()
+  })
+
+  it("formats valid date range in en-AU text format", () => {
```

Comment:
Wouldn't this be the responsibility of the translation library?

Reply:
Yes :P It's more just testing two to ensure we actually have some i18n going since we were somewhat home-rolling it. We only require the consumer to pass in a string, and then we choose a locale object on their behalf :)